### PR TITLE
Tko element kws index

### PR DESCRIPTION
**Pull request by tkoukkari**
Added support for using index argument with xpaths

### CODE REVIEW

File: QWeb/internal/util.py [hunk 1]
```diff
@@ -310,3 +311,11 @@ def escape_xpath_quotes(text):
     if '"' in text:
         return f"'{text}'"
     return f'"{text}"'
+
+
+def anchor_to_index(anchor):
```
Comment (turunenm) [inline QWeb/internal/util.py]: typing should be added to function definition

**Review by turunenm — CHANGES_REQUESTED** [review]
Add typing to commented function and this one's good to go

File: QWeb/internal/util.py (hunk 1 above)
Comment (tkoukkari) [reply QWeb/internal/util.py]: Oops, yep, will do.

Comment (tkoukkari) [reply QWeb/internal/util.py]: done with the latest commit

**Review by turunenm — APPROVED** [review]
lgtm

**Review by irosengr — APPROVED** [review]

**Pull request merged**